manifests/thanos-query-deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
         - --log.format=logfmt
         - --query.replica-label=prometheus_replica
         - --query.replica-label=rule_replica
-        - --store=dnssrv+_grpc._tcp.thanos-store.thanos.svc.cluster.local
+        - --store=dnssrv+_grpc._tcp.thanos-receive-ingestor-default.thanos.svc.cluster.local:10901
```

Comment:
Why this has gotten overwritten? Shouldn't be just an addition? We still want to read from the store, right?

Reply:
The problem here is that the `example.jsonnet` file does not deploy successfully to a cluster 'off-the-bat' - which is a pretty bad user experience.

The reason is that the store component fails to start up because it is missing object storage secrets. I think we should either:
1. Not deploy store as part of  `example.jsonnet`
2. Deploy minio & correctly configure store to pull from there & receive to ship there?

Reply:
We discussed this, and we don't intend `examples` to work out-of-the-box. They are just examples. If you really want to see examples at work maybe we can configure them with a filesystem bucket. We should keep it simple.

Reply:
Updates `example.jsonnet` to include both store & ingestor endpoints.